### PR TITLE
Team Page -- Modal

### PR DESCRIPTION
**Pull request by MCurran16**
+ Add Member Button

### CODE REVIEW

File: assets/css/views/team-details/team-details.css.styl [hunk 1]
```diff
@@ -208,6 +214,176 @@ roster-remove-margin-top = (roster-user-container-height / 2) - (24px / 2)
...
+	color steel-gray
+	font-primary-light-custom(16px)
+
+::-webkit-input-placeholder
+	text-transform uppercase
+
+::-moz-placeholder
+	text-transform uppercase
+	
+:-ms-input-placeholder
```
Comment (hunterbrowning) [inline assets/css/views/team-details/team-details.css.styl]: you could do all these like 
```
::-webkit-input-placeholder,
::-moz-placeholder,
:-ms-input-placeholder
   text-transform uppercase
```

couldn't you?

Comment (MCurran16) [inline assets/css/views/team-details/team-details.css.styl]: Hmm, I tried that earlier in development and it didn't work. Now it's doing the job for me! 

fixed

Comment (MCurran16) [inline assets/css/views/team-details/team-details.css.styl]: It does not work. I tried it

**Comment by hunterbrowning** [comment]
LGTM

**Pull request merged**